src/scenes/Home/index.jsx

```diff
@@ -136,7 +136,10 @@ const Home = () => {
             with a common goal of innovation for social good!
           </p>
           <h3>JOIN US</h3>
-          <p>Follow us on Twitter (@HackIllinois) and Instagram (@HackIllinois) to be notified of our event updates!</p>
+          <p>
+            Follow us on Twitter (<a href="https://twitter.com/hackillinois/">@HackIllinois</a>) and Instagram
```

Comment:
Is it possible to make the links appear blue like normal links? Or is that a default thing from the browser

Reply:
blue and underlined*

Reply:
I got rid of the default style because the blue links are hard to read against that background, but I'll add the underline back

Reply:
Makes sense, sounds good